### ISSUE TITLE
can't create Matrix block in Supertable in except English

### ISSUE THREAD

**Issue by mersy** [engram-design/SuperTable#278]
### Description
I can't create new block on matrix in Supertable.
So, I found this issue, and can create new matrix block when switching language to English.

> Can’t add blocks to matrix inside Supertable after initial creation · Issue #154 · verbb/super-table
> https://github.com/verbb/super-table/issues/154

Always I used craft cms in Japanese.

This issue is closed 2.0.8 fixed this problem, but I used 2.1.20.
Is that issue and problem is fixed or not in except English?

### Steps to reproduce

1. Add a matrix field to a Supertable
1. Add some blocks to the matrix
1. Save the Supertable field
1. Edit the Supertable field again
1. Add new blocks to the matrix inside the Supertable
1. Save the Supertable field -> will generate the error

### Additional info

- Plugin version: 2.1.20.
- Craft version: 3.1.30

**Comment by engram-design** [comment]
I take it your site is a multi-site? Did you create the fields from scratch in the current Craft and Super Table plugin version, or on a previous version of the plugin?

I'm wondering if you might be able to put a video together to illustrate the issues? Sorry if that's a hassle, but I've just tried to replicate it on my end without much luck.

**Comment by mersy** [comment]
@engram-design
Thank you for your comment.
My site is multi-site. And I created this site on March 2019. Sorry, I forgot Super Table version when installing it, maybe later 2.0.8.

And I captured my cp.

In Japanese NG
<img width="551" alt="japanese01_00" src="https://user-images.githubusercontent.com/205165/59346723-e2e27180-8d4d-11e9-8a79-a02314da4035.png">

![japanese01_01](https://user-images.githubusercontent.com/205165/59349317-4ec7d880-8d54-11e9-9f12-36d4850ce820.gif)


In English OK
![english](https://user-images.githubusercontent.com/205165/59346720-e249db00-8d4d-11e9-9790-9b8f80852050.gif)

In Japanese NG
![japanese02_01](https://user-images.githubusercontent.com/205165/59346719-e249db00-8d4d-11e9-9cc6-ad63fb715139.gif)

In this case, I can create matrix field in English.
But I'll talk about this case to my client, so I just want to know that issue is fixed or not.
